test/JDBC/input/BABEL_GRANT_CONNECT-vu-prepare.sql

```diff
@@ -0,0 +1,12 @@
+create database grant_connect_db1;
+go
+
+use grant_connect_db1;
+go
+
+create table grant_connect_t1(a int);
+go
+
+create login grant_connect_abc with password = 'Babel123'
+go
+
```

Comment:
Do we need to switch back to master db at the end of each test? I think the question is more about, will JDBC start a new session for each test script, both for normal JDBC and during upgrade?

Reply:
No, we don't need to switch back to master at the end of the test. 
Yes, JDBC starts a new session for each test script.